stl/inc/limits

```diff
@@ -1135,14 +1152,18 @@ _NODISCARD constexpr int _Countr_zero(const _Ty _Val) noexcept {
 
 template <class _Ty, enable_if_t<_Is_standard_unsigned_integer<_Ty>, int> _Enabled = 0>
 _NODISCARD _CONSTEXPR20 int _Popcount(const _Ty _Val) noexcept {
-#if _HAS_POPCNT_INTRINSICS
+#if _HAS_POPCNT_INTRINSICS || _HAS_CNT_INTRINSICS
 #if _HAS_CXX20
     if (!_STD is_constant_evaluated())
 #endif // _HAS_CXX20
     {
+#if _HAS_POPCNT_INTRINSICS
         return _Checked_x86_x64_popcount(_Val);
+#elif _HAS_CNT_INTRINSICS
+        return _Arm64_popcount(_Val);
+#endif

```

Comment:
As this is nested preprocessor logic, I think that a comment on the `#endif` would be desirable.